src/Web/Sfa.Das.Sas.Web/Views/Provider/ProviderDetails.cshtml

```diff
@@ -0,0 +1,132 @@
+﻿@using Sfa.Das.Sas.Web.Extensions
+@using Sfa.Das.Sas.Web.ViewModels
+@model ProviderDetailsViewModel
+
+@{
+    ViewBag.Title = "Provider Details";
+    ViewBag.Description = "";
+}
+<main id="content" class="provider-detail">
+    <div class="grid-row">
+        <div class="column-two-thirds">
+            <h1 class="heading-xlarge" id="provider-name">
+                @Model.ProviderName 
+                @if (@Model.NationalProvider)
+                {
+                    <span class="tag tag-national">National</span>
+                }
+            </h1>
+        </div>
+    </div>
+    <div class="grid-row">
+        <div class="column-two-thirds">
+            <dl class="data-list--provider data-list--compact">
+                <dt>Trading names:</dt>
+                <dd>@Model.TradingNames</dd>
+                <dt>UKPRN:</dt>
+                <dd>@Model.Ukprn</dd>
+            </dl>
+        </div>
+        <div class="column-one-third">
+            <div class="panel--contact">
+                <h2 class="heading-medium">Contact details</h2>
+                <dl>
+                    @GetStandardPropertyAsLinkHtml("Website:", "course-link", "apprenticeshipContactTitle", "apprenticeshipContact", Model.Website, Model.ProviderName + " website")
```

Comment:
These methods are 1 to 1 with their usage, so you could either remove parameters so they're lighter (and update the method names) or remove the methods to expose the conditional logic